config/locales/en/claims/schools/users.yml

```diff
@@ -17,7 +17,7 @@ en:
           page_title: Check your answers
           title: Check your answers
           caption: Add user
-          add_user: Add user
+          add_user: Save user
```

Comment:
Should we rename the key here?

I'd also push for context based keys rather than content based keys. In this case, it would probably be `submit` or along those lines.